Corzbank.Data.Entities/Models/UserModel.cs

```diff
@@ -1,10 +1,28 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 using System.Text;
 
-namespace Corzbank.Data.Entities
+namespace Corzbank.Data.Entities.Models
 {
     public class UserModel
     {
+        [Required(ErrorMessage ="Firstname is required")]
+        public string Firstname { get; set; }
+
+        [Required(ErrorMessage = "Lastname is required")]
+        public string Lastname { get; set; }
+
+        [Required(ErrorMessage = "PhoneNumber is required")]
+        public string PhoneNumber { get; set; }
+
+        [Required(ErrorMessage = "Email is required")]
+        public string Email { get; set; }
+
+        [Required(ErrorMessage = "Password is required")]
+        public string Password { get; set; }
+
+        [Required(ErrorMessage = "ConfirmPassword is required")]
+        public string ConfirmPassword { get; set; }
```

Comment:
Password and ConfirmPassword should match, add that validation rule

Reply:
resolved

Reply:
where? I don't see it here